components/combobox/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@spectrum-css/combobox",
+  "version": "1.0.0-alpha.0",
+  "description": "The Spectrum CSS combobox component",
+  "license": "Apache-2.0",
+  "author": "Adobe",
+  "homepage": "https://opensource.adobe.com/spectrum-css/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git",
+    "directory": "components/combobox"
+  },
+  "bugs": {
+    "url": "https://github.com/adobe/spectrum-css/issues"
+  },
+  "main": "dist/index-vars.css",
+  "scripts": {
+    "build": "gulp"
+  },
+  "peerDependencies": {
+    "@spectrum-css/pickerbutton": "^2.0.0",
+    "@spectrum-css/popover": "^5.0.0 || ^6.0.0",
+    "@spectrum-css/tokens": "^7.5.0"
+  },
+  "devDependencies": {
+    "@spectrum-css/component-builder-simple": "^2.0.5",
+    "@spectrum-css/icon": "^3.0.34",
```

Comment:
I don't think there's a need to bring in icon separately since it's being used as a part of the pickerbutton (which will bring it in as it's own dependency).

Reply:
I've gone ahead and removed this in [73d7557](https://github.com/adobe/spectrum-css/pull/1556/commits/73d75570b78e9d501e251f441d86e696022778d5). Though I am wondering; should textfield also have `@spectrum-css/icon` as a dependency, like pickerbutton does? It doesn't currently. It uses icons for invalid and valid. Unless that's an opt-in thing.